src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -92,13 +102,37 @@ export const SueReportLocation = ({
         locations={locations}
         mapCenterPosition={mapCenterPosition}
         mapStyle={styles.map}
+        isMyLocationButtonVisible
+        onMyLocationButtonPress={() =>
+          Alert.alert(texts.sue.report.alerts.hint, texts.sue.report.alerts.myLocation, [
+            {
+              text: texts.sue.report.alerts.no,
+              onPress: () => null
+            },
+            {
+              text: texts.sue.report.alerts.yes,
+              onPress: () => {
+                const location = position || lastKnownPosition;
+
+                if (location) {
+                  setUpdatedRegion(true);
+                  setSelectedPosition(location.coords);
+                  reverseGeocode(location.coords);
+                }
+              }
+            }
+          ])
+        }
         onMapPress={({ nativeEvent }) => {
-          setSelectedPosition({
-            ...nativeEvent.coordinate
-          });
-
+          setUpdatedRegion(true);
+          setSelectedPosition(nativeEvent.coordinate);
           reverseGeocode(nativeEvent.coordinate);
         }}
+        updatedRegion={
+          selectedPosition && updatedRegion
+            ? { ...selectedPosition, latitudeDelta: 0.01, longitudeDelta: 0.01 }
+            : undefined
+        }
```

Comment:
This part of the code causes the map to be centred and the zoom to change for each pin added. Do we want this to happen? Maybe we can remove the `latitudeDelta` and `longitudeDelta` values?

Reply:
please try with removing the deltas

Reply:
I tried it and the zoom didn't change. Because we already have a zoom value set before. So I'm deleting it.

|before choosing a location|after with detas value|after without detas value|
|--|--|--|
![Simulator Screenshot - iPhone 14 Pro Max - 2024-01-30 at 12 08 21](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/d273b0e3-0464-492c-8e8b-9faf4a8c55ad)|![Simulator Screenshot - iPhone 14 Pro Max - 2024-01-30 at 12 08 44](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/e58256a1-d64c-4370-a07a-cd5956fde479)|![Simulator Screenshot - iPhone 14 Pro Max - 2024-01-30 at 12 08 59](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/5dac81b7-eb18-4ac3-a91b-ea2b704bff4a)|

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/937e0d0d7ff2eee80a4ea1a53ea2c175473bdc16